Add MaterialColors property format to docs (#140)
rojo-rbx/rojo.space: docs/properties.md

}
```

### MaterialColors

For both implicit and explicit values, the type is an object that looks like this:

```json
{
	"$properties": {

> 		"MaterialColors": {

Reviewer: We usually have `"ImplicitExample"` and `"ExplicitExample"` to make it clear which is which. Here they are both called `"MaterialColors"` which breaks our norm and makes it a little confusing (especially since this is technically invalid as it contains duplicate keys).

			"Grass": [10, 20, 30],
			"Asphalt": [40, 50, 60],
			"LeafyGrass": [255, 155, 55]
		},
		"MaterialColors": {
			"MaterialColors": {
				"Grass": [10, 20, 30],
				"Asphalt": [40, 50, 60],
				"LeafyGrass": [255, 155, 55]
			}
		}
	}
}
```

> The key-value pairs within the object should be [Material](https://create.roblox.com/docs/reference/engine/enums/Material) enum items mapped to 3-item arrays representing the RGB of the specified color. This format does not support decimals values. Only the values you wish to change the defaults of need to be specified.

Reviewer: I think we can make the type info more concise without losing clarity.
```suggestion
The key-value pairs within the object should be [Material](https://create.roblox.com/docs/reference/engine/enums/Material) enum items mapped to arrays of 3 integers representing the RGB of the specified color. Only the values you wish to change the defaults of need to be specified.
```
